Tools/autotest/arducopter.py

```diff
@@ -12311,6 +12311,22 @@ def ScriptingFlipMode(self):
         # Test done
         self.land_and_disarm()
 
+    def RTLYaw(self):
+        '''test that vehicle yaws to original heading on RTL'''
+        self.set_parameters({
+            'WP_YAW_BEHAVIOR': 3,  # 3 is WP_YAW_BEHAVIOR_LOOK_AHEAD
```

Comment:
Thanks for this.  I think WP_YAW_BEHAVIOR = 1 (Face next waypoint) could be a better choice because this is closer to the default behaviour which is 2 (Face next waypoint except RTL)

Reply:
Ah ,well.  Why not both? :-)